src/utils/Api.js

```diff
@@ -0,0 +1,74 @@
+class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getAppInfo() {
+    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
+  }
+
+  _checkResponse(res) {
+    return res.ok ? res.json() : Promise.reject(`Error : {error.status}`);
+  }
+
+  getUserInfo() {
+    return fetch(`${this._baseUrl}/users/me`, {
+      headers: this._headers,
+    });
```

Comment:
* The avatar and profile data are not displayed on the page https://snipboard.io/ahJ2Pl.jpg 


`.then(this._checkResponse);` should be here to parse the response


```js

  getUserInfo() {
    return fetch(`${this._baseUrl}/users/me`, {
      headers: this._headers,
    }).then(this._checkResponse);
  }
```